setup.py

```diff
@@ -1,10 +1,3 @@
 from setuptools import setup
 
-setup(
-    name="hipscat-config",
-    author="Sam Wyatt",
-    url="https://github.com/astronomy-commons/lsd2",
-    setup_requires=['setuptools_scm', 'setuptools_scm_git_archive'],
-    packages=["hipscat"],
-    include_package_data=True
-)
+setup(setup_requires=["setuptools_scm"])
```

Comment:
is the `setup_requires=["setuptools_scm"]` necessary here? I think because setuptools_scm is already listed in the pyproject.toml we might be able to get away with just doing `setup()` :) not a blocking thing tho

Reply:
Wheel is not happy with removing the `setup_requires` argument here.